dev/benchmarks/macrobenchmarks/test/cull_opacity_perf_e2e.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The test should be run as:
+// flutter drive -t test/cull_opacity_perf_e2e.dart --driver test_driver/e2e_test.dart --trace-startup --profile
+
+import 'package:macrobenchmarks/common.dart';
+
+import 'util.dart';
+
+Future<void> main() async {
```

Comment:
This file probably belongs to `test_driver` as this is unlikely to be used as a unit test.

Reply:
Same reason above.